src/topics/badge/badge.api.ts

```diff
@@ -0,0 +1,93 @@
+import { FastifyInstance, FastifyRequest } from "fastify";
+import { AttesterNetwork } from "../attester";
+import { serializeBadgeApiType } from "./badge.api.helpers";
+
+type BadgeQueryType = FastifyRequest<{
+  Params: {
+    network: string;
+  };
+}>;
+
+type NetworkBadgeQueryType = FastifyRequest<{
+  Params: {
+    network: string;
+    badgeId: string;
+  };
+}>;
+
+const routes = async (fastify: FastifyInstance) => {
+  /**
+   * Returns all the badges from a specific network.
+   * @returns 404 when the Badges or the Network are not found, 200 otherwise with the BadgeAPIType[]
+   */
+  fastify.get(
+    "/badges/:network/badges.json",
+    async (req: BadgeQueryType, res) => {
+      const { network } = req.params;
+
+      const badges = Object.keys(fastify.attesters)
+        .filter((attesterName) =>
+          fastify.attesters[attesterName].hasNetworkConfiguration(
```

Comment:
why hasNetworkConfiguration and not just `network in networkConfigurations` ?

Reply:
Didn't knew about this syntax, thanks